source/guides/1.0/getting-started.md

```diff
@@ -1031,7 +1029,7 @@ resources :books, only: [:index, :new, :create]
 ```
 
 To get a sense of what routes are defined, now we've made this change, you can
-use the special command-line task `routes` to inspect the end result:
+run `bundle exec hanami routes` on your command-line to inspect the end result:
```

Comment:
Hmm, the actual command is below. IMHO it's better to keep the original wording, since it makes the user acquainted with what a "task" means.